.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+
+- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-cloud-ruby/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
+- [ ] Follow the instructions in [CONTRIBUTING](CONTRIBUTING.md). Most importantly, ensure the tests and linter pass by running `bundle exec rake ci` in the gem subdirectory.
```

Comment:
@dazuma Looking at [CONTRIBUTING.md#tests](https://github.com/googleapis/google-cloud-ruby/blob/master/.github/CONTRIBUTING.md#tests) just now, I noticed that we omit `bundle exec` from the `rake` command examples. I think this dates from long ago when we were much more agnostic about using Bundler. Should I update those usages in this PR?

Reply:
Done.